tests/js/session-tests.js

```diff
@@ -413,98 +413,42 @@ module.exports = {
             }).then(() => TestCase.assertTrue(progressCalled));
     },
 
-    /*
-    testProgressNotificationsForRealmOpenAsync() {
+    testClientReset() {
+        // FIXME: try to enable for React Native
         if (!platformSupported) {
             return;
         }
 
-        const username = Utils.uuid();
-        const realmName = Utils.uuid();
-
-        const credentials = Realm.Sync.Credentials.nickname(username);
-        return runOutOfProcess(__dirname + '/download-api-helper.js', username, realmName, REALM_MODULE_PATH)
-            .then(() => Realm.Sync.User.login('http://127.0.0.1:9080', credentials))
-            .then(user => {
-                return new Promise((resolve, reject) => {
-                    let config = {
-                        sync: {
-                            user,
-                            url: `realm://127.0.0.1:9080/~/${realmName}`
-                        },
-                        schema: [{ name: 'Dog', properties: { name: 'string' } }],
-                    };
+        const partition = Utils.genPartition();
+        let creds = Realm.Credentials.anonymous();
+        let app = new Realm.App(appConfig);
+        return app.logIn(creds).then(user => {
+            return new Promise((resolve, _reject) => {
+                var realm;
```

Comment:
```suggestion
                let realm;
```

Minor thing, but there's no need to introduce or old friend `var` here :)